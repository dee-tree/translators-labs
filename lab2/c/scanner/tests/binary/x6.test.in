111111111111111111111111111111111xb