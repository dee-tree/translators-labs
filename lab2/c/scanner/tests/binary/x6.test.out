
Fatal error: number 111111111111111111111111111111111xb is too big
